drivers/hmis/app/models/hmis/unit_occupancy.rb

```diff
@@ -29,7 +29,16 @@ class Hmis::UnitOccupancy < Hmis::HmisBase
   has_one :occupancy_period, class_name: 'Hmis::ActiveRange', as: :entity, dependent: :destroy
   # Service record that relates to this occupancy (likely a BedNight or BedNight-ish custom service)
   belongs_to :hmis_service, class_name: 'Hmis::Hud::HmisService', optional: true
-  accepts_nested_attributes_for :occupancy_period
+
+  # To discuss: accepts_nested_attributes_for has some trickiness that we weren't accounting for, so I'm proposing to remove it.
```

Comment:
Let's discuss this more. I think another alternative would be to keep `accepts_nested_attributes_for` and add the `update_only` option, but I haven't tested that.

(I'll remove this code comment after the discussion)